pkg/otlp/model/translator/metrics_translator_test.go

```diff
@@ -354,6 +354,33 @@ func TestMapIntMonotonicWithReboot(t *testing.T) {
 	)
 }
 
+func TestMapIntMonotonicReportFirstValue(t *testing.T) {
+	values := []int64{10, 15, 20}
+	slice := pmetric.NewNumberDataPointSlice()
+	slice.EnsureCapacity(len(values))
+
+	startTs := int(getProcessStartTime()) + 1
+	for i, val := range values {
+		point := slice.AppendEmpty()
+		point.SetStartTimestamp(seconds(startTs))
+		point.SetTimestamp(seconds(startTs + i))
+		point.SetIntValue(val)
+	}
+
+	ctx := context.Background()
+	tr := newTranslator(t, zap.NewNop())
+	consumer := &mockTimeSeriesConsumer{}
+	tr.mapNumberMonotonicMetrics(ctx, consumer, exampleDims, slice)
```

Comment:
nit: Despite what other tests do, I think we should write this in terms of `MapMetrics` instead if it's not too much hassle, this will make it easier to refactor code without rewriting tests

You can also do this later when you rewrite the tests in terms of JSON, up to you!